utils/config/chain.config.ts

```diff
@@ -30,7 +30,7 @@ const chainPropertyMap: Config<ChainProperties> = {
   bsx: toChainProperty(
     10041,
     12,
-    'BSX',
+    'KSM',
     'https://calamar.play.hydration.cloud/basilisk/search?query='
```

Comment:
just saw this link here, maybe we want to switch this up for https://basilisk.subscan.io/? 
ref:
- https://github.com/kodadot/nft-gallery/issues/3294

Reply:
I would take a look

Reply:
Done